README.md

```diff
@@ -20,27 +22,30 @@ Next, let's define a reducer function in `reducers/counterReducer.js` to pass to
 
 ```javascript
 // src/reducers/counterReducer.js
-export default function counterReducer(state=0, action){
+const counterReducer = (state = 0, action) => {
   switch (action.type) {
     case 'INCREMENT_COUNT':
-      return state + 1;
+      return state + 1
     case 'DECREMENT_COUNT':
-      return state - 1;
+      return state - 1
     case 'RESET_COUNT':
-      return 0;
+      state = 0
```

Comment:
Why set `state = 0` here? This shouldn't have to be here I don't think. Just returning 0 should modify the state in the store like in all the other cases in this switch